src/pages/vue/quickstart.md

```diff
@@ -372,23 +372,24 @@ On our main `IonFab`, we are setting its positioning with the vertical and horiz
 Now let's wire up a click handler to this. When clicking the FAB button, we want to navigate to a new page (which we will create in a moment). To do this, we will need to get access to Vue Router's navigation API. This can be done by importing `useRouter` from the `vue-router` package.
 
 ```html
-import { add } from 'ionicons/icons';
+<template>
+  <ion-content>
```

Comment:
Sorry, I forgot to mention we need to wrap `ion-content` in an `ion-page` as well.